sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md

```diff
@@ -89,6 +96,24 @@ rename-mapping:
   Scope: BenefitRecommendationScope
   Grain: BenefitRecommendationUsageGrain
   OperatorType: ComparisonOperatorType
+  DismissAlertPayload.properties.closeTime: -|datetime
+  DismissAlertPayload.properties.statusModificationTime: -|datetime
+  Alert.properties.modificationTime: -|datetime
+  Alert.properties.creationTime: -|datetime
+  Alert.properties.statusModificationTime: -|datetime
+  View.properties.scope: -|arm-id
+  ScheduledAction.properties.scope: -|arm-id
+  ScheduledAction.properties.viewId: -|arm-id
+  ExportDeliveryDestination.resourceId: DestinationId|arm-id
```

Comment:
consider keeping the name as it is, `Destination` is implied in the class name already, duplicated in the property name.